Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -211,6 +211,12 @@ public static function initializeConfiguration(Bootstrap $bootstrap)
             $configurationManager->refreshConfiguration();
         }
 
+        // Manually inject settings into the PackageManager as the package manager is excluded from the proxy class building
+        $flowSettings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Flow');
+        if (is_array($flowSettings)) {
```

Comment:
If setting a for some reason is empty and we end up not injecting anything, will that leave the packageManager in a similar state with no settings, since it's excluded (as commented above this code)?
So if the settings is not an array, should we pass a empty array to ensure that eh injectSettings method is being called so the packagemanager get its setting a property set no matter what?